applications/app/controllers/InteractiveController.scala

```diff
@@ -68,121 +64,90 @@ class InteractiveController(
     capiLookup.lookup(path, range = Some(ArticleBlocks))
   }
 
-  def itemResponseToModel(item: Future[ItemResponse])(implicit
```

Comment:
This method was replaced as it's doing too much:

* it accepts a Future[ItemResponse] when ItemResponse should be enough
* it shouldn't be doing both model conversion and API/wider error handling